chasm/lib/scheduler/scheduler.go

```diff
@@ -121,7 +134,19 @@ func CreateScheduler(
 		req.FrontendRequest.InitialPatch,
 	)
 
-	// TODO - use visibility component to update SAs
+	// Update visibility with custom attributes.
+	visibility, err := sched.Visibility.Get(ctx)
+	if err != nil {
+		return nil, nil, ErrUnprocessable
```

Comment:
I would use internal status errors here but this is also fine because unknown errors get translated to that by default AFAIR.